### ISSUE TITLE
Enable rotate popup suggestion?

### ISSUE THREAD

**Issue by lsnch** [TeamNewPipe/NewPipe#8135]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Hi, I love your app :)

The button seems to be disabled for newpipe. Every other app has it, it's a really nifty little thing!

![photo5353006960540236231](https://user-images.githubusercontent.com/20382674/161405469-5f42141d-3759-4832-80e4-a4cc3032bacc.jpg)

### Why do you want this feature?

I hate auto-rotate, and newpipe always gets it wrong which orientation I want, so I have to go to shade and enable-disable auto-rotate to change orientation.

**Comment by SameenAhnaf** [comment]
Are you asking this for main player only? Then, it is a duplicate of https://github.com/TeamNewPipe/NewPipe/issues/4500

**Comment by lsnch** [comment]
The main player can already be rotated with the `[]` button. But the search screen, comments, history have no way to go portrait after watching a video. The background player has the button, but not the main app. It'd nice to have a universal way to do this.

**Comment by opusforlife2** [comment]
> newpipe always gets it wrong which orientation I want

I'm assuming you want to go back to portrait after watching a video. Try this. After the video ends and the Replay button appears, use the Back button or the Full Screen button to go back to portrait mode.

**Comment by SameenAhnaf** [comment]
>But the search screen, comments, history have no way to go portrait after watching a video.

Yeah, https://github.com/TeamNewPipe/NewPipe/issues/5967 should be the default behavior. Is that what you want?

**Comment by lsnch** [comment]
>I'm assuming you want to go back to portrait after watching a video. Try this. After the video ends and the Replay button appears, use the Back button or the Full Screen button to go back to portrait mode.

No only then, but I can't remember exact scenarios. It has to do with switching between video and background, among other things. What you suggest is a workaround, like toggling rotate in notification shade. To me, the popup button feels infinitely more seamless.

PS. I've looked into this a bit more. It seems the popup button works as long as you don't press `[]`. If you do, the app needs to be restarted. There must be some sort of flag connected to `[]` that disables popup.

**Comment by SameenAhnaf** [comment]
I don't get it. Why should the popup player be related after finishing a video? Are you willing to watch video and use other apps at the same time? Go to `Settings`>`Video and Audio`>`Minimize on App Switch`> Select `Minimize to popup player`

**Comment by cbiere** [comment]
It really looks like the full screen button tells Android to enforce landscape mode for the app like games usually do. Thanks for the tip though. By ignoring the full screen button and just using the manual rotation button the issue is far less annoying.

**Comment by lefuturiste** [comment]
what is the [] button, I don't have it

**Comment by locuturus** [comment]
If you have a large DPI setting, or use a tablet, the rotation problem is even worse. 

Starting with what works (and remember I'm testing with 600+ DPI):

1. Starting from portrait, if you rotate the phone and use the phone's rotation suggestion then newpipe rotates, but adopts a multi-column tablet UI so the video is not full screen. Rotating the phone back to portrait does prompt the system's rotate suggestion and so this works. So far this seems good to me.

Now we get into the problems that I face:

2. Starting from portrait, if you tap on [   ] the video does not go full screen. Instead, NewPipe will enter the landscape multi column tablet UI as above, but now it's locked there*. Tapping [   ] again will actually start full screen. The full screen video player can rotate now but you can't return to the main app in portait* 

3. If you rotate the app to landscape using the system suggestion, as in 1., you get landscape tablet UI and you have the option to tap [   ]. If you do so, you get full screen video but you are locked to landscape app UI*.

*You can enable the system's auto rotate and then NewPipe rotates as expected without locking. This is not a good workaround though because very often users do not want autorotation in general.

**Comment by ctntt** [comment]
> what is the [] button, I don't have it

The [] button is only displayed, when Auto-Rotate is turned off.
